samples/AddDefaultNetworkPolicy.md

```diff
@@ -21,6 +21,9 @@ spec:
         kinds:
         - Namespace
         name: "*"
+        selector:
+          matchLabels:
+               nirmata.io/auto-config: "enabled"
```

Comment:
I think we want these generate policies in best practice triggers on namespace creation. This label is specific to nirmata configuration, we don't want to add it to best practice.